package.json

```diff
@@ -48,7 +48,7 @@
     "npm-release": "node scripts/npm-release.js",
     "prestart": "yarn ensure-deps",
     "start": "cypress open --dev --global",
-    "stop-only": "npx stop-only --skip .cy,.publish,.projects,node_modules,dist,dist-test,fixtures,lib,bower_components,src,__snapshots__ --exclude e2e.ts,cypress-tests.ts,*only_spec.js",
+    "stop-only": "npx stop-only --skip .cy,.publish,.projects,node_modules,dist,dist-test,fixtures,lib,bower_components,src,__snapshots__ --exclude e2e.ts,cypress-tests.ts,*only.cy.js",
```

Comment:
Do `e2e.ts` & `cypress-tests.ts` still exist or have they also been updated to use `.cy.ts`?

Reply:
- `cypress-tests.ts` exists, but this is not actually a test executed using Cypress. It uses `dtslint`. For this reason, I don't think it should use a `.cy.ts` extension.
- I cannot see `e2e.ts` in this branch, or anywhere else. I think it does not exist, and hasn't for a while. I removed the reference to it.